docs/_integration-with-existing-apps-ios.md

```diff
@@ -0,0 +1,605 @@
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+## Key Concepts
+
+The keys to integrating React Native components into your iOS application are to:
```

Comment:
Maybe we can also mention that this guide covers both old and new architecture? And if you want to use new arch you can install pods with a flag?

Reply:
this will be new arch only. All the references are to 0.76 and it will be published with that.
It follows the usual guidelines to disable it, eventually. But this is out of the scope of this guide, imho.